plugins/catalog-backend-module-msgraph/src/microsoftGraph/config.test.ts

```diff
@@ -72,7 +65,7 @@ describe('readMicrosoftGraphConfig', () => {
         tenantId: 'tenantId',
         clientId: 'clientId',
         clientSecret: 'clientSecret',
-        authority: 'https://login.example.com',
+        authority: 'https://login.example.com/',
```

Comment:
Does this slash actually matter? Makes me wonder, if somebody out there has config with no slash in it, will some behavior be different now? :) If that's the case, I'd worry about that

Reply:
I've tested with both and they both work fine.

@azure/identity constants exclude trailing slash, so if anything I'd be more worried about the urls with trailing slashes not working. - https://github.com/Azure/azure-sdk-for-js/blob/037feadfe9fb35b442bc1403ccefd136dc57e377/sdk/identity/identity/src/constants.ts#L22-L39